compute_tools/src/bin/fast_import.rs

```diff
@@ -120,8 +303,18 @@ pub(crate) async fn main() -> anyhow::Result<()> {
         (None, None)
     };
 
-    // Get source connection string either from S3 spec or direct argument
-    let source_connection_string = if let Some(s3_prefix) = &args.s3_prefix {
+    let superuser = "cloud_admin";
+    let pg_port = || {
+        args.pg_port.unwrap_or_else(|| {
+            info!("pg_port not specified, using default 5432");
+            5432
+        })
+    };
```

Comment:
nit: You can specify default values in the `Args` struct definition:
```
63     #[clap(long, default_value_t = 5432)]
64     pg_port: u16, // port to run postgres on, 5432 is default
```

Reply:
indeed, fixed